pkgs/top-level/all-packages.nix

```diff
@@ -617,7 +617,11 @@ with pkgs;
 
   dec-decode = callPackage ../development/tools/dec-decode { };
 
-  dnf4 =  with python3Packages; toPythonApplication dnf4;
+  dnf-plugins-core =  with python3Packages; toPythonApplication dnf-plugins-core;
+
+  dnf4-unwrapped =  with python3Packages; toPythonApplication dnf4-unwrapped;
+
```

Comment:
```suggestion
```
since we have the package in python3Packages already, we don't need that or can expose it via passthru as unwrapped